examples/demo-stf/src/runtime.rs

```diff
@@ -53,12 +55,13 @@ use sov_value_setter::query::{ValueSetterRpcImpl, ValueSetterRpcServer};
 
 #[cfg_attr(
     feature = "native",
-    cli_parser(DefaultContext),
+    cli_parser(DefaultContext, "sequencer"),
```

Comment:
I see there is `// #[cli_skip]` bellow. Can we di it this way instead of hardcoding module name here?

Reply:
so the thing is - The original method i tried was #[cli_skip] and it worked, but the problem is cargo hack fails when the native feature is not passed. for some reason, restricting cli_skip to also work only for native doesn't seem to work.

#[cfg_attr(feature = "native", cli_skip)]
seems to be ignored and compiler complains that Sequencer doesn;t have the necessary traits implemented.

additionally, i also tried defining cli_skip as a no-op proc macro, but it still doesn't seem to work.

I only did the second attribute to cli_parser after trying for a lot of time to fix the errors for #[cli_skip] with cargo hack. I think its fine if native is supplied, but in other cases, it's causing failures.

Reply:
I see two approaches:
1. Can you check what serde is doing? I think some crates derive serde for they datatypes only when serde feature is enabled (and they can use field attributes)
2. If `CallMessage` is not specified for a module, the type defaults to `pub enum NonInstantiable {}` we could potentially derive the clip machinery for it as well.

We should explore option1 first since it is less invasive on the `sov-modules-api`

Reply:
This uses an attribute now 👍